content/en/template.json

```diff
@@ -53,5 +53,9 @@
         "url": "https://medium.com/feed/@iojs"
       }
     ]
+  },
+  "deprecation": {
+    "title": "io.js has merged with the Node.js project again.",
+    "content": "There won't be any further separate io.js releases, all of io.js' features are available in Node.js v4 and above.<br>{{link 'Visit nodejs.org' 'https://nodejs.org'}} {{link 'Download latest Node.js version' 'https://nodejs.org/en/download/'}}"
```

Comment:
Nit: Comma after `releases` should be a period and `all of io.js' features` should be the start of a new sentence.

Nit #2: `All of the features in io.js` rather than `All of io.js' features`.